time_execution/decorator.py

```diff
@@ -69,6 +69,10 @@ def after(self):
         if origin:
             metric['origin'] = origin
 
+        use_resource_uri = self.params.get('use_resource_uri', False)
```

Comment:
In my personal opinion, this metric is too specific.
`timeexecution` itself doesn't know anything about REST, SOAP or other technologies. It just measures execution time for a wrapped function.

And you can easily extend metrics sent by `timeexecution` library with custom hook, so I don't see any reason for this change.

Reply:
As discussed, I agree with Pavel, in this case the hooks (or the custom hooks) should be enough.